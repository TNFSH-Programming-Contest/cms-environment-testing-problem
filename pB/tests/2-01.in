-268298120 399057441
-399488028 -475205199
