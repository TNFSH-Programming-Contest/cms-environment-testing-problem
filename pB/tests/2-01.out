-667786148 -76147758
